pkgs/applications/science/misc/megam/default.nix

```diff
@@ -12,11 +12,17 @@ stdenv.mkDerivation {
 
   patches = [ ./ocaml-includes.patch ./ocaml-3.12.patch ];
 
-  buildInputs = [ ocaml ncurses ];
+  strictDeps = true;
 
-  nativeBuildInputs = [ makeWrapper ];
+  nativeBuildInputs = [ makeWrapper ocaml ];
 
-  makeFlags = [ "CAML_INCLUDES=${ocaml}/lib/ocaml/caml" ];
+  buildInputs = [ ncurses ];
+
+  makeFlags = [
+    "CAML_INCLUDES=${ocaml}/lib/ocaml/caml"
+    "WITHBIGARRAY=bigarray.cma"
+    "WITHUNIX=unix.cma"
```

Comment:
The original `WITHBIGARRAY` includes `-lbigarray`, which [should be dropped](https://github.com/bcpierce00/unison/commit/23bf53e5c28d5e0b42735c6da36385f3d1de8d36) since OCaml 4.08. The third is indeed not necessary, thanks! Would you like me to convert it to a patch?

Reply:
No it is fine. My comment was only for the removed line.